x-pack/plugins/security_solution/server/lib/detection_engine/rule_types/indicator_match/threat_mapping/create_threat_signals.ts

```diff
@@ -222,7 +222,7 @@ export const createThreatSignals = async ({
 
         // push warning message to appear in rule execution log
         results.warningMessages.push(
-          `maxClauseCount error received from elasticsearch (${errorType})`
+          `maxClauseCount error received from elasticsearch (${errorType}), setting IM rule page size to ${maxClauseCountValue}`
```

Comment:
@dhurley14 

would it also show error message with clauses number? I think it would be useful
currently, it just says "maxClause is set to", w/o number. Looks like error message is trimmed
 
![image](https://github.com/elastic/kibana/assets/92328789/22e6f4a1-1336-42f1-a966-18b40f40f3a7)

Reply:
ah yeah this would be helpful - I'll update it.

Reply:
Okay I think I got it now. Let me know if this looks right:

<img width="1196" alt="Screen Shot 2024-04-29 at 10 20 11 AM" src="https://github.com/elastic/kibana/assets/915763/939478ea-38cb-40cb-89e6-27b06144c91f">

Reply:
look good! thanks